python/doc/theory/probabilistic_modeling/trend_transform.rst

```diff
@@ -0,0 +1,62 @@
+Trend computation
+-----------------
+
+A multivariate stochastic process
+:math:`X: \Omega \times\cD \rightarrow \Rset^d` of dimension :math:`d`
+where :math:`\cD \in \Rset^n` may write as the sum of a trend function
+:math:`f_{trend}: \Rset^n \rightarrow \Rset^d` and a stationary
+multivariate stochastic process
+:math:`X_{stat}: \Omega \times\cD \rightarrow \Rset^d` of dimension
+:math:`d` as follows:
+
+.. math::
+  :label: tsDecomposition
+
+    \forall \omega \in \Omega, \, \forall \vect{t} \in \cD, \,X(\omega,\vect{t}) = X_{stat}(\omega,\vect{t}) + f_{trend}(\vect{t})
+
+The objective here is to identify the trend function :math:`f_{trend}`
```

Comment:
I propose to shorten the end of the  text : we only need to mention that the library allows the identification of a trend function from a realization of a process (give the link to the TrendFactory class where the algorithm is detailed).